app/src/main/java/io/appwrite/services/Account.java

```diff
@@ -20,6 +24,7 @@ public Account(Client client){
     /*
      * Get currently logged in user data as JSON object.
      */
+    @RequiresApi(api = Build.VERSION_CODES.R)
```

Comment:
@punitdiwan what is this line used for?

Reply:
Here @RequiresApi - Denotes that the annotated element should only be called on the given API level or higher.

So I don't think this to be merged